app/client/src/components/shared/LocationMap.js

```diff
@@ -1155,6 +1157,10 @@ function LocationMap({ layout = 'narrow', windowHeight, children }: Props) {
       });
     });
 
+    // turn off clustering if there are 20 or less stations
+    monitoringLocationsLayer.featureReduction =
+      graphics.length > 20 ? monitoringClusterSettings : undefined;
```

Comment:
One small point: I see that you initialized the layer with the `monitoringClusterSettings`, but in the case where there are 20 or fewer graphics, I'd suggest that it's better practice to set the property to `null` rather than `undefined`.
```suggestion
      graphics.length > 20 ? monitoringClusterSettings : null;
```

Reply:
Good catch! I'll go ahead and make that change.